app/assets/javascripts/mobile.js

```diff
@@ -18,23 +18,29 @@ $(function () {
       if (this.$navbar.closest('aside').css('float') !== 'none') {
         return;
       }
-
       this.setup();
     },
     control : function (e) {
       var $this = $(e.target),
-          instance = this;
+          instance = this,
+          hasAnalytics = GOVUK.analytics && GOVUK.analytics.trackEvent;
 
       if(!$this.hasClass('show-all-parts-open')) {
         $this.addClass('show-all-parts-open');
         $this.text($this.text().replace(/Show/, 'Hide'));
         instance.$navbar.removeClass('page-navigation-closed');
         instance.$navbar.addClass('page-navigation-open');
+        if hasAnalytics {
```

Comment:
Consistency aside, It's not valid JavaScript: you end up with 'Uncaught SyntaxError: Unexpected identifier'.